extension/android/jni/jni_layer_llama.cpp

```diff
@@ -202,12 +202,38 @@ class ExecuTorchLlmJni : public facebook::jni::HybridClass<ExecuTorchLlmJni> {
           data_files_vector,
           executorch::extension::Module::LoadMode::MmapUseMlockIgnoreErrors);
       std::string decoder_model = "llama3"; // use llama3 for now
```

Comment:
Replace the model name here accordingly, available path is https://github.com/pytorch/executorch/blob/f80916e8efc046d56f85625b271a5a0016f9424a/examples/qualcomm/oss_scripts/llama/runner/runner.cpp#L121